src/main/java/seedu/address/model/country/Country.java

```diff
@@ -87,9 +87,12 @@ public boolean equals(Object other) {
         // state check
         Country c = (Country) other;
 
-        return countryCode.equals(c.countryCode)
-                && countryName.equals(c.countryName)
-                && countryNotes.equals(c.countryNotes);
+        return countryCode.equals(c.countryCode);
+    }
+
+    @Override
+    public String toString() {
+        return countryCode;
     }
```

Comment:
Initially it was because `ClientUtil.java` needed to build the command given the client's details, I changed the implementation of `getClientDetails` to explicitly use `getCountryCode()` so this function isn't currently in use now. I thought it would make sense to have `countryCode` since that is the value passed into the object like the other fields.

Reply:
Yeah, but usually the purpose of the toString is to return a more "human readable" format of the object, so maybe countryName would be a better choice for the return value.